features/src/main/feature/camel-features.xml

```diff
@@ -477,6 +477,15 @@
         <bundle dependency='true'>wrap:mvn:com.amazonaws/aws-xray-recorder-sdk-core/${aws-xray-version}</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-aws-xray/${project.version}</bundle>
     </feature>
+    <feature name='camel-azure-files' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-ftp</feature>
+        <bundle dependency='true'>mvn:org.apache.camel.karaf/camel-cloudevents/${project.version}</bundle>
```

Comment:
here I am using camel-ftp instead of core as it is needed by the component's bundle. Should I still use `camel-cloudevents` as a feature?

Reply:
yes

Reply:
fixed